test/test.js

```diff
@@ -65,6 +65,24 @@ describe('static methods', function () {
     });
   });
 
+  describe('#fromString()', function () {
+    var originalVec, vec;
+
+    before(function () {
+      originalVec = new Vector(100, 200);
+      vec = Vector.fromString(originalVec.toString());
+    });
+
+    it('should return an instance of Vector', function () {
+      expect(vec).to.be.an.instanceof(Vector);
+    });
+
+    it('should have axis from object', function () {
+      expect(vec).to.have.property('x', originalVec.x);
+      expect(vec).to.have.property('y', originalVec.y);
```

Comment:
Why not just deep equal with the original? Or, on the other hand, maybe we should not use `toString` in this test at all and just use `x:100, y:200` here? So that it's explicit how it's supposed to work.